README.md

```diff
@@ -1,6 +1,180 @@
-# Developing methods to quantify drug response heterogeneity
+# tHMM
 
 [![codecov](https://codecov.io/gh/meyer-lab/tHMM/branch/master/graph/badge.svg)](https://codecov.io/gh/meyer-lab/tHMM)
 [![Documentation Status](https://readthedocs.org/projects/tHMM/badge/?version=latest)](https://lineage-growth.readthedocs.io/en/latest/?badge=latest)
 
 [Manuscript Build](https://meyer-lab.github.io/tHMM/manuscript.html)
+
+`tHMM` is a Python package for developing methods to quantify drug response heterogeneity, using single-cell data in the form of cell lineages.
+
+- [Overview](#Overview)
+- [Documentation](#Documentation)
+- [Systems Requirements](#system-requirements)
+- [Installation Guide](#Installation-Guide)
+- [Demo Instructions for Use](#Demo)
+
+# Overview
+
+`tHMM` is an open-source Python package that implements an Expectation-Maximization algorithm for a hidden Markov model to simultaneously solve for hidden states and model parameters. The purpose of this model is to identify phenotypic heterogeneity among cancer cells exposed to different concentrations of a drug, and cluster the cells based on various observations, most importantly, taking to account the cell-cell reationship in decision making. The model takes in experimental observations in the form of binary lineages of single cells in a specific format in Excel sheets (please refer to lineage/data/heiser_data/new_version to see the excel sheets for each condition). Currently, the experimental observations include the single cell fate, and cell cycle phase durations, but the model is flexible with no limitations of how many observations to use, so long as they are heritable. The model has been tested on synthetic data, and also on experimental data of AU565 breast cancer cells exposed to lapatinib and gemcitabine doses. This framework properly accounts for time and fate censorship, as the experiments run for a finite amount of time. For a thorough tutorial of the implemented method, please refer to `manuscript/05.methods.md`.
+
+# Documentation
+The `docs` folder includes a few tutorials for getting started with the package. All the functions should have a docstring explaining the purpose of the function, as well as the inputs, outputs, and the type of the variables used.
+
+# System Requirements
+## Hardware requirements
+`tHMM` package requires only a standard computer with enough RAM to support the in-memory operations.
+
+## Software requirements
+
+### OS requirements
+This package is supported for *macOS*, *Windows*, and *Linux*. The package has been tested on the following systems:
+- macOS: Mojave (10.14.1)
+- Linux: Ubuntu 20.04
+- Windows: 10
+
+### Python dependencies
+`tHMM` mainly depends on the Python scientific stack.
```

Comment:
I think you can say it requires `virtualenv`, then everything else can be installed by `make venv`.

Reply:
done.